docs/reference/commandline/ps.md

```diff
@@ -51,7 +51,47 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
 
-## Successfully exited containers
+
+#### Label
+
+The `label` filter allows to filter container based on the presence of the label and its value.
+
+    $ docker ps --filter "label=color"
+    CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
+    673394ef1d4c        busybox             "top"               47 seconds ago      Up 45 seconds                           nostalgic_shockley
+    d85756f57265        busybox             "top"               52 seconds ago      Up 51 seconds                           high_albattani
+
+
+This will display all containers that have a `color` label, no matter what value it has.
+
+    $ docker ps --filter "label=color=blue"
+    CONTAINER ID        IMAGE               COMMAND             CREATED              STATUS              PORTS               NAMES
+    d85756f57265        busybox             "top"               About a minute ago   Up About a minute                       high_albattani
+
+
+This will list all running containers with a color label of blue.
+
+#### Name
+
+The `name` filter allows filter container based on their name.
+
+    $ docker ps --filter "name=nostalgic_stallman"
+    CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
+    9b6247364a03        busybox             "top"               2 minutes ago       Up 2 minutes                            nostalgic_stallman
+
+This will display all containers that have a name that matches `nostalgic_stallman`.
```

Comment:
The following matches all containers with a name containing the  `nostalgic_stallman` string.